aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -216,6 +216,7 @@ void cpu_upsample_nearest_channels_last(
     }
   };
 
+  TORCH_INTERNAL_ASSERT(channels > 0);
```

Comment:
This should be TORCH_CHECK, because it is triggered by user providing invalid data. `TORCH_INTERNAL_ASSERT` is for behavior when there is an internal error in pytorch, the message there is "File a bug for pytorch", this is not what should happen here.

Reply:
Thanks for the heads up! Fixed.